test/cases/include_next.c

```diff
@@ -3,6 +3,7 @@
 #include_next <stdalign.h>
```

Comment:
Oh I read that this was also including `my_include.h`, that suggestion doesn't make much sense without it, it was supposed to test that `#include` and `#include_next` do the same thing when used from the main file.

Reply:
I see, I can add a test for that behavior too